components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -524,6 +526,19 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                                     {openLinkLabel}
                                 </button>
                             </div>
+                            {!useLatest && (
+                                <Alert type="info" className="mt-4 w-96">
+                                    You can change the default editor for opening workspaces in{" "}
```

Comment:
**thought**: This alert could become obsolete or less useful so we could remove it in the future if the actions were clearer like in https://github.com/gitpod-io/gitpod/issues/6910 or more flexiible as seen in the design exploration mentioned in https://github.com/gitpod-io/gitpod/pull/8783#discussion_r828522647. Re-posting the design exploration for visibility. Feedback is always welcome!

<img width="955" alt="dropdown" src="https://user-images.githubusercontent.com/120486/163410339-12752cf8-a8bb-4b50-996f-68f83e74b949.png">